include/joker.h

```diff
@@ -83,14 +122,14 @@ void joker_destroy(Joker **joker);
 
 // Unique effects like "Four Fingers" or "Credit Card" will be hard coded into game.c with a conditional check for the joker ID from the players owned jokers
 // game.c should probably be restructured so most of the variables in it are moved to some sort of global variable header file so they can be easily accessed and modified for the jokers
-JokerEffect joker_get_score_effect(Joker *joker, Card *scored_card);
+JokerEffect joker_get_score_effect(Joker *joker, Card *scored_card, int scored_when);
```

Comment:
```suggestion
JokerEffect joker_get_score_effect(Joker *joker, Card *scored_card, enum JokerCallback joker_event);
```

If you're passing the enum you made, make it the parameter type.
Also suggesting the `scored_when` to `joker_event` rename.

Reply:
What do you mean by making it the parameter type?